config/site/site.config.js

```diff
@@ -123,6 +123,9 @@ const siteConfig = {
         gitHubUrl: npmPackage.repository.url.replace('git+', ''),
       },
     },
+
+    /* The default inclusion of test evidence for the site. 'true' is the default value. */
+    includeTestEvidence: true,
```

Comment:
Should this be at the root level of the `siteConfig` obj instead of the  `appConfig`? Other config for pages & routes are at the `siteConfig` level. Also, this would allow us to open up a testEvidence config obj later on if we wanted to enhance